captioning/detect_caption.py

```diff
@@ -0,0 +1,46 @@
+from .utils import get_detectron_features
+from pythia.common.sample import Sample, SampleList
+import gc
+import torch
+
+# Wrapper class over Detection and Captioning model that defines the
+# combined the caption generation from raw image path
```

Comment:
can't parse this sentence

Reply:
:laughing:  sorry, it reads: 

# Wrapper class over Detection (feat-extraction) and Captioning model, that defines `predict` function which takes in the imagepath and generates the caption.

Reply:
Fixing this in the latest commit.